unst_msh_gen/ocn_ww3.py

```diff
@@ -63,10 +68,10 @@ def create_msh():
     # solve |dh/dx| constraints in spacing
     jigsawpy.cmd.marche(opts, spac)
     
-    opts.mesh_file = "mesh_50km_nobc.msh"
+    opts.mesh_file = "uglo_100km_BlkS.msh"
     
     opts.hfun_scal = "absolute"
-    opts.hfun_hmax = +50.           # uniform at 30.km
+    opts.hfun_hmax = 100.           # uniform at 30.km
```

Comment:
Please provide a note here about how this variable should be defined and what it means.  Additional descriptions of the other opts. variables would also be appreciated.